.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+sudo: false
+script: mvn -Pdev verify
+jdk:
+  - oraclejdk8
+  - openjdk8
+env:
+  global:
+    - JAVA_OPTS="-Xmx1024M -Xms512M"
+notifications:
+  irc:
+    channels:
+      - "irc.freenode.org#jena"
```

Comment:
I just took that from [here](https://jena.apache.org/help_and_support/#other-channels) in our docs. I hang out in the Jena IRC room all the time and nothing ever gets notified into there. I thought it would be convenient (it's great for other projects on which I work), but it's hardly of the essence. I can take it out if it worries you. What is "jarvis"?